application/src/main/java/org/opentripplanner/gtfs/mapping/IdFactory.java

```diff
@@ -12,17 +13,27 @@ class IdFactory {
   private final String feedId;
 
   IdFactory(String feedId) {
-    this.feedId = feedId;
+    this.feedId = Objects.requireNonNull(feedId);
   }
 
   /** Map from GTFS to OTP model, {@code null} safe. */
-  @Nullable
   FeedScopedId createId(@Nullable AgencyAndId id) {
     return id == null ? null : new FeedScopedId(feedId, id.getId());
   }
```

Comment:
Why did you remove the `@Nullable` annotation from this?

Reply:
That was a mistake.